specification/hybridconnectivity/resource-manager/readme.md

```diff
@@ -101,7 +101,6 @@ This is not used by Autorest itself.
 ``` yaml $(swagger-to-sdk)
 swagger-to-sdk:
   - repo: azure-sdk-for-net-track2
-  - repo: azure-sdk-for-python
```

Comment:
Since Python could be generated from typespec now, there is no need keep configuration for swagger.